hawkbit-repository/hawkbit-repository-api/src/main/java/org/eclipse/hawkbit/repository/model/Target.java

```diff
@@ -99,4 +100,8 @@ default TargetWithActionType getTargetWithActionType() {
      */
     boolean isRequestControllerAttributes();
 
+    /**
+     * @return attributes provided by the controller
+     */
+    Map<String, String> getControllerAttributes();
```

Comment:
I'm not sure if this is needed. Same functionality is available in org.eclipse.hawkbit.repository.TargetManagement.getControllerAttributes(String)